### ISSUE TITLE
Indenting the injected products json parameters

### ISSUE THREAD

**Issue by idomic** [ploomber/ploomber#877]
When ploomber injects the parameters from the yaml file into the notebook, those are single lined dictionaries.

We can ease it for the user and indent it so it's readable for more than a few parameters.

**Comment by GaoyangLiu** [comment]
More context of this problem, output of `pip freeze`:
absl-py==1.1.0
aiohttp==3.8.1
aiosignal==1.2.0
ansiwrap==0.8.4
antlr4-python3-runtime==4.8
anyio==3.6.1
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.5
async-timeout==4.0.2
attrs==21.4.0
autocfg==0.0.8
autogluon==0.4.2
autogluon-contrib-nlp==0.0.1b20220208
autogluon.common==0.4.2
autogluon.core==0.4.2
autogluon.features==0.4.2
autogluon.tabular==0.4.2
autogluon.text==0.4.2
autogluon.vision==0.4.2
autopep8==1.6.0
Babel==2.10.3
backcall==0.2.0
backoff==1.11.1
beautifulsoup4==4.11.1
bleach==5.0.0
blis==0.7.7
boto3==1.24.12
botocore==1.27.12
cachetools==5.2.0
catalogue==2.0.7
catboost==1.0.6
certifi==2022.6.15
cffi==1.15.0
charset-normalizer==2.0.12
click==8.1.3
cloudpickle==2.1.0
colorama==0.4.5
contextvars==2.4
cycler==0.11.0
cymem==2.0.6
dask==2021.11.2
debugpy==1.6.0
decorator==5.1.1
defusedxml==0.7.1
Deprecated==1.2.13
distributed==2021.11.2
entrypoints==0.4
et-xmlfile==1.1.0
executing==0.8.3
fairscale==0.4.6
fastai==2.5.6
fastcore==1.4.5
fastdownload==0.0.6
fastjsonschema==2.15.3
fastprogress==1.0.2
filelock==3.7.1
flake8==4.0.1
fonttools==4.33.3
frozenlist==1.3.0
fsspec==2022.5.0
gitdb==4.0.9
GitPython==3.1.27
gluoncv==0.10.5.post0
google-auth==2.8.0
google-auth-oauthlib==0.4.6
graphviz==0.20
greenlet==1.1.2
grpcio==1.46.3
HeapDict==1.0.1
huggingface-hub==0.7.0
humanize==4.2.0
idna==3.3
imageio==2.19.3
immutables==0.18
importlib-metadata==4.11.4
ipdb==0.13.9
ipykernel==6.15.0
ipython==7.34.0
ipython-genutils==0.2.0
jedi==0.17.2
Jinja2==3.0.3
jmespath==1.0.1
joblib==1.1.0
json5==0.9.8
jsonschema==4.6.0
jupyter-client==7.3.4
jupyter-core==4.10.0
jupyter-server==1.17.1
jupyter-server-mathjax==0.2.5
jupyter-tabnine==1.2.3
jupyterlab==3.4.3
jupyterlab-git==0.37.1
jupyterlab-pygments==0.2.2
jupyterlab-server==2.14.0
jupytext==1.13.8
kiwisolver==1.4.3
langcodes==3.3.0
lightgbm==3.3.2
locket==1.0.0
loguru==0.6.0
Markdown==3.3.7
markdown-it-py==2.1.0
MarkupSafe==2.1.1
matplotlib==3.5.2
matplotlib-inline==0.1.3
mccabe==0.6.1
mdit-py-plugins==0.3.0
mdurl==0.1.1
mistune==0.8.4
mlxtend==0.20.0
monotonic==1.6
msgpack==1.0.4
multidict==6.0.2
murmurhash==1.0.7
nbclassic==0.3.7
nbclient==0.6.4
nbconvert==6.5.0
nbdime==3.1.1
nbformat==5.4.0
nest-asyncio==1.5.5
networkx==2.8.4
notebook==6.4.12
notebook-shim==0.1.0
nptyping==1.4.4
numpy==1.22.4
oauthlib==3.2.0
omegaconf==2.1.2
opencv-python==4.6.0.66
openpyxl==3.0.10
packaging==21.3
pandas==1.3.5
pandocfilters==1.5.0
papermill==2.3.4
parso==0.7.1
partd==1.2.0
pathy==0.6.1
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.0.1
ploomber==0.19.6
ploomber-scaffold==0.3.1
plotly==5.8.2
portalocker==2.4.0
posthog==1.4.9
preshed==3.0.6
prometheus-client==0.14.1
prompt-toolkit==3.0.29
protobuf==3.20.1
psutil==5.8.0
ptyprocess==0.7.0
pure-eval==0.2.2
pyarrow==8.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.8.0
pycparser==2.21
pydantic==1.8.2
pyDeprecate==0.3.2
pyflakes==2.4.0
Pygments==2.12.0
pygraphviz @ file:///home/conda/feedstock_root/build_artifacts/pygraphviz_1654467256069/work
pyparsing==3.0.9
pyrsistent==0.18.1
python-dateutil==2.8.2
pytorch-lightning==1.6.4
pytz==2022.1
PyWavelets==1.3.0
PyYAML==6.0
pyzmq==23.2.0
ray==1.10.0
redis==4.3.3
regex==2022.6.2
requests==2.28.0
requests-oauthlib==1.3.1
rsa==4.8
s3transfer==0.6.0
sacrebleu==2.1.0
sacremoses==0.0.53
scikit-image==0.19.3
scikit-learn==1.0.2
scipy==1.7.3
seaborn==0.11.2
Send2Trash==1.8.0
sentencepiece==0.1.95
six==1.16.0
smart-open==5.2.1
smmap==5.0.0
sniffio==1.2.0
sortedcontainers==2.4.0
soupsieve==2.3.2.post1
spacy==3.3.1
spacy-legacy==3.0.9
spacy-loggers==1.0.2
SQLAlchemy==1.4.37
sqlparse==0.4.2
srsly==2.4.3
stack-data==0.3.0
tabulate==0.8.9
tblib==1.7.0
tenacity==8.0.1
tensorboard==2.9.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
terminado==0.15.0
textwrap3==0.9.2
thinc==8.0.17
threadpoolctl==3.1.0
tifffile==2022.5.4
timm==0.5.4
tinycss2==1.1.1
tokenizers==0.12.1
toml==0.10.2
toolz==0.11.2
torch==1.10.2
torchmetrics==0.7.3
torchvision==0.11.3
tornado==6.1
tqdm==4.64.0
traitlets==5.3.0
transformers==4.16.2
typer==0.4.1
typing_extensions==4.2.0
typish==1.9.3
urllib3==1.26.9
wasabi==0.9.1
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.3.3
Werkzeug==2.1.2
wrapt==1.14.1
xgboost==1.4.2
yacs==0.1.8
yarl==1.7.2
zict==2.2.0
zipp==3.8.0

**Comment by GaoyangLiu** [comment]
`pip install black` will fix it in my case.

**Comment by edublancas** [comment]
cool, closing this
